src/app/waste-water/waste-water-report/input-data/input-data.component.html

```diff
@@ -146,7 +135,7 @@ <h4 class="show-print">Assessment Input Data</h4>
       </tr>
       <tr>
         <td [ngClass]="{'indicate-report-field-different': checkActivateSludgeItemDifferent('Biomass') == true}">
-          Biomass Yield
+          VSS/TSS Ratio of Biomass
```

Comment:
If you haven't already, check with Kristina that these two refer to the same thing.